layout/sparse/sparse.go

```diff
@@ -0,0 +1,56 @@
+package sparse
+
+import (
+	"log"
+
+	v1 "github.com/google/go-containerregistry/pkg/v1"
+	"github.com/google/go-containerregistry/pkg/v1/empty"
+
+	"github.com/buildpacks/imgutil/layout"
+
+	"github.com/buildpacks/imgutil"
+)
+
+var _ imgutil.Image = (*Image)(nil)
+
+// Image is a struct created to overrides the Save() method of the image.
+// a sparse Image is saved on disk but does not include any layers in the `blobs` directory.
+type Image struct {
+	layout.Image
+}
+
+// NewImage returns a new Image saved on disk that can be modified
+func NewImage(path string, from v1.Image) (*Image, error) {
+	img, err := layout.NewImage(path, layout.FromBaseImage(from))
+	if err != nil {
+		return nil, err
+	}
+
+	image := &Image{
+		Image: *img,
+	}
+	return image, nil
+}
+
+func (i *Image) Save(additionalNames ...string) error {
```

Comment:
I think implementing `SaveAs` is simple and probably worth doing now so we don't run into any issues using the `Image` interface. This goes for `layout.Image` as well. Rebase is going to use `SaveAs` once [this pr](https://github.com/buildpacks/lifecycle/pull/985) lands.

```suggestion
func (i *Image) Save(additionalNames ...string) error {
	return i.SaveAs(i.Name(), additionalNames...)
}

func (i *Image) SaveAs(name string, additionalNames ...string) error {
```

Reply:
Yeah, I did this change, my mistake the method I want to leave non-implemented is **SaveFile()**